client/components/lists/listBody.js

```diff
@@ -140,4 +175,77 @@ BlazeComponent.extendComponent({
       keydown: this.pressKey,
     }];
   },
+
+  onCreated() {
+    dropdownMenuIsOpened = false;
+  },
+
+  onRendered() {
+    const $textarea = this.$('textarea');
+    const currentBoard = Boards.findOne(Session.get('currentBoard'));
+    $textarea.textcomplete([
+      // User mentions
+      {
+        match: /\B@(\w*)$/,
+        search(term, callback) {
+          callback($.map(currentBoard.members, (member) => {
+            const username = Users.findOne(member.userId).username;
+            return username.indexOf(term) === 0 ? username : null;
+          }));
+        },
+        template(value) {
+          return value;
+        },
+        replace(username) {
+          return `@${username} `;
+        },
+        index: 1,
+      },
+
+      // Labels
+      {
+        match: /\B#(\w*)$/,
+        search(term, callback) {
+          callback($.map(currentBoard.labels, (label) => {
+            const labelName = (!label.name || label.name === '')
```

Comment:
I believe that even if a blue label has a name we should still display it in the autocomplete list. If there are both an unnamed blue label, and a few blue labels with names with should display the unnamed label first and named label after that (in no particular order).